### PR TITLE
feat/deployment

### PR DESCRIPTION
**Pull request by dayt47**
fly.io deployment with Docker

### CODE REVIEW

File: Agenda-client/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# syntax = docker/dockerfile:1
+
+# Adjust NODE_VERSION as desired
+ARG NODE_VERSION=20.9.0
+FROM node:${NODE_VERSION}-slim as base
+
+LABEL fly_launch_runtime="Vite"
+
+# Vite app lives here
+WORKDIR /app
```
[the remark this answers is not in the document]
Comment (itzMaffi) [reply Agenda-client/Dockerfile:10]: Ok that's the build directory so maybe yes

**Comment by itzMaffi** [comment]
Did we test if this works as expected? Is there a way to test?

File: .github/workflows/fly.yml
Comment (dinarabs) [inline .github/workflows/fly.yml:1]: there is 2 yml files with same name, which one we need?
also i will be adding CI part, i will merge deployment to CI part

Comment (itzMaffi) [reply .github/workflows/fly.yml:1]: There should be only one github workflow folder in the root, I think something went wrong with the commits

Comment (dayt47) [reply .github/workflows/fly.yml:1]: there should be 1 "fly.yml" in .github/workflows and 1 "fly.toml" in the root directory

Comment (dinarabs) [reply .github/workflows/fly.yml:1]: taking account we will have server side later, app root is `/Agenda-client` or should we create root folder hold both client and server folders?

Comment (dayt47) [reply .github/workflows/fly.yml:1]: the root is not agenda-client, agenda client is **in** the root folder. if we need a server we just create a folder besides the agenda client so they are both in the root folder

Comment (dinarabs) [reply .github/workflows/fly.yml:1]: @dayt47 pls then remove one of them, there is still two github workflows.

Comment (dinarabs) [reply .github/workflows/fly.yml:1]: @dayt47 re root folder: i forgot there is `CW-thesis-project` folder :D 

also there are two .toml files: one under root and other inside `/Agenda-client`, can you please remove latter as well?

Comment (itzMaffi) [reply .github/workflows/fly.yml:1]: Somethings feel wrong tho, we have two identical dockerfiles for the client, one of which is in the root folder which imo is not correct. In general, even if we are in the root folder we only want to build deploy what's in the Agenda-client.

Comment (dayt47) [reply .github/workflows/fly.yml:1]: let me check it locally

**Review by itzMaffi — APPROVED** [review]
lgtm

**Pull request merged**